qiskit/primitives/base/estimator_task.py

```diff
@@ -0,0 +1,92 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+
+"""
+Estiamtor Task class
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import Tuple, Union
+
+import numpy as np
+
+from qiskit import QuantumCircuit
+
+from .base_task import BaseTask
+from .bindings_array import BindingsArray, BindingsArrayLike
+from .observables_array import ObservablesArray, ObservablesArrayLike
+from .shape import ShapedMixin
+
+
+@dataclass(frozen=True)
+class EstimatorTask(BaseTask, ShapedMixin):
+    """Task for Estimator.
+    Task is composed of triple (circuit, observables, parameter_values).
+    """
+
+    observables: ObservablesArray
+    parameter_values: BindingsArray = BindingsArray(shape=())
```

Comment:
`bindings_array` might be better?

Reply:
I think `parameter_values` is more intuitive, especially for people don't just pass in an array instead of creating a `BindingsArray` explicitly.